public/locales/en.json

```diff
@@ -293,6 +294,7 @@
   "utilization": "Utilization",
   "veSdl": "veSDL",
   "veSdlCalculator": "veSDL calculator",
+  "veSdlNetworkText": "Looks like you are in {{chainName}} network.\n veSDL is only available on the Ethereum Mainnet!",
```

Comment:
are we using Sdl or SDL for keys?

Reply:
I think SDL will be good. But considering camel case, I used `veSdl` and this problem is what I wanted to ask before.
Will I change all `veSdl` to `veSDL`?

Reply:
Because our UI is using veSDL, so I recommend to use veSDL  than veSdl.